assets/templates/app/drivers/device.js

```diff
@@ -1,8 +1,8 @@
 'use strict';
 
-const Homey = require('homey');
+import { Device } from 'homey';
 
-class MyDevice extends Homey.Device {
+class MyDevice extends Device {
```

Comment:
this is a .js file, we can't use import here (yet)

Reply:
OK some merges went wrong here